activerecord/lib/active_record/relation/finder_methods.rb

```diff
@@ -352,6 +352,10 @@ def exists?(conditions = :none)
     # compared to the records in memory. If the relation is unloaded, an
     # efficient existence query is performed, as in #exists?.
     def include?(record)
+      # current implementation primarily expects an Active Record instance to be passed as `record`
+      # we can be sure we won't find anything if non-AR object is passed
```

Comment:
```suggestion
      # The existing implementation relies on receiving an Active Record instance as the input parameter named record. 
      # Any non-Active Record object passed to this implementation is guaranteed to yield no results.
```

Reply:
Thank you! I hope you don't mind that I changed the last bit to "guaranteed to return `false`". Since "no results" is not particularly fair and might be interpreted as `nil`

Reply:
That's totally fine 👍